CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# Release 2.36.0 25-04-2022
+
+- Upgraded Envoy to 1.22.0 (#684)
+- Added option to enable strict request validation in transcoder (#685)
+- Fixed a test-only race condition by protecting critical sections (#683)
+- Updated flag --service help text (#681)
+- Updated outdated flag --version help text (#680)
```

Comment:
remove line 5, 6, 7 as they are not user visible

Reply:
With these 3 lines removed, it looks like the CHANGELOG only has the following remaining?

- Upgraded Envoy to 1.22.0 (#684)
- Added response gzip,brotli compression (#675)

Reply:
Removed.